content/en/docs/reference/using-api/health-checks.md

```diff
@@ -19,12 +19,12 @@ The `healthz` endpoint is deprecated (since Kubernetes v1.16), and you should us
 The `livez` endpoint can be used with the `--livez-grace-period` [flag](/docs/reference/command-line-tools-reference/kube-apiserver) to specify the startup duration.
 For a graceful shutdown you can specify the `--shutdown-delay-duration` [flag](/docs/reference/command-line-tools-reference/kube-apiserver) with the `/readyz` endpoint.
 Machines that check the `healthz`/`livez`/`readyz` of the API server should rely on the HTTP status code.
-A status code `200` indicates the API server is `healthy`/`live`/`ready`, depending of the called endpoint.
-The more verbose options shown below are intended to be used by human operators to debug their cluster or specially the state of the API server.
+A status code `200` indicates the API server is `healthy`/`live`/`ready`, depending on the called endpoint.
+The more verbose options shown below are intended to be used by human operators to debug their cluster or specify the state of the API server.
```

Comment:
I don't think the verbose options help "specify" the state of the API server.
They help us understand the state of the API server and therefore, this could be rewritten as,

The more verbose options shown below are intended to be used by human operators to debug their cluster or understand the state of the API server.

WDYT?

Reply:
I agree. I was too focused on choosing words that resembled the original spelling.

Is it OK to use the GitHub suggestion feature or should I commit&push a new commit?
```suggestion
The more verbose options shown below are intended to be used by human operators to debug their cluster or understand the state of the API server.
```

Reply:
Either way is fine, however I'd recommend squashing & merging the commit before I give an lgtm because it's tidier.

Reply:
squashed on af2a60b57ffa2b30a7a39e15be6b351dfee4bcf2